src/Containers/DistrictContainer/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import React from 'react';
+import DistrictCard from '../../Components/DistrictCard';
+import { getDistricts } from '../../utils/api';
+import type { DetailedDistrictType } from '../../types';
+
+export class DistrictContainer extends React.Component {
+  state: {
+    districts: Array<DetailedDistrictType>,
+  };
+
+  constructor() {
+    super();
+    this.state = { districts: [] };
```

Comment:
I'm not sure what you mean by "outside"

Are you talking about:
```
 state: {
     districts: Array<DetailedDistrictType>,
   };
```

on line 8?